src/main/scala/vectorpipe/osm/internal/ElementToFeature.scala

```diff
@@ -322,16 +322,13 @@ private[vectorpipe] object ElementToFeature {
 
         val (a, b) = t.splitAt(i)
 
-        /* Hand-hold the typechecker */
-        val res: Option[(Line, ElementData, Int, Vector[OSMLine])] =
-          Some((line, h.data, i, a ++ b.tail))
-
         /* Return early */
-        return res.pure[IntState]
+        return Some((line, h.data, i, a ++ b.tail))
```

Comment:
Evidently `dotty` will make it so that these extra parens are a thing of the past (at least for function application)

Reply:
In this case I actually want the quadruplet to be visible.